blink.js

```diff
@@ -1 +1,8 @@
-// YOUR CODE GOES HERE
+function blink(obj){
+  obj.toggle();
+}
+
+$.fn.blink = function(interval) {
+  window.setInterval(blink, interval, this);
+  return this;
+};
```

Comment:
One advantage of using a self-evaluating function around all of this code (as [suggested](https://learn.jquery.com/plugins/basic-plugin-creation/#protecting-the-alias-and-adding-scope)) is that you avoid creating a global `blink` variable.